HISTORY.rst

```diff
@@ -14,4 +14,9 @@ Release History
 0.1.2 (2019-07-08)
 -------------------
 
-- Minor updates, many bug fixes, no breaking changes.
\ No newline at end of file
+- Minor updates, many bug fixes, no breaking changes.
+
+0.1.3 (2019-08-09)
+-------------------
+
+- Minor updates, many bug fixes, no breaking changes.
```

Comment:
Actually we have breaking changes (oef-sdk etc.). We should probably be more explicit here about what this brings: bump of oef sdk and compatibility changes, better debugging, monitoring and documentation.